### PR TITLE
Reset all circuit playground components

### PR DESCRIPTION
**Pull request by ajpal**
~Do not submit until https://github.com/code-dot-org/chrome-serial-app/pull/2 is in. That PR fixes the remaining issues with sending too much at once across the wire. This change requires sending a bunch of commands across the wire to reset the state, so we need to fix the flakiness before submitting this.~

This is my attempt at resetting all state on each component. It was a little hard to track down some of the state, so if you can think of anything I may have missed please let me know! In particular, I'm a little worried that the sensors may still be sending readings across the wire, but I'm not sure how to check for this.

### CODE REVIEW

**Comment by joshlory** [comment]
Merged https://github.com/code-dot-org/chrome-serial-app/pull/2 and the new version (33) is [now published to the Chrome Web Store](https://chrome.google.com/webstore/detail/codeorg-serial-connector/ncmmhcpckfejllekofcacodljhdhibkg?hl=en).

image

File: apps/src/lib/kits/maker/PlaygroundComponents.js [hunk 1]
```diff
@@ -69,6 +69,44 @@ export function createCircuitPlaygroundComponents(board) {
...
+  }
+
+  if (components.tempSensor) {
+    components.tempSensor.disable();
+  }
+
+  if (components.accelerometer) {
+    components.accelerometer.stop();
+  }
+}
```
Comment (islemaster) [inline apps/src/lib/kits/maker/PlaygroundComponents.js]: This seems correct.  It'd be nice to deduplicate this code with `destroyCircuitPlaygroundComponents`, below.

**Review by islemaster — APPROVED** [review]
:+1:

**Comment by islemaster** [comment]
Oh - tests please?

**Pull request merged**